src/Form/Form.js

```diff
@@ -0,0 +1,64 @@
+import "./Form.css";
+import React, { useState } from "react";
+
+export default function Form() {
+  const [meme, setMeme] = React.useState({
+    topText: "",
+    bottomText: "",
+    randomImage: "http://i.imgflip.com/1bij.jpg",
+  });
+  const [allMemes, setAllMemes] = React.useState([]);
+  React.useEffect(() => {
+    fetch("https://api.imgflip.com/get_memes")
+      .then((res) => res.json())
+      .then((data) => setAllMemes(data.data.memes));
+  }, []);
+  function getMemeImage() {
+    const randomNumber = Math.floor(Math.random() * allMemes.length);
+    const url = allMemes[randomNumber].url;
+
+    setMeme((prevMeme) => ({
+      ...prevMeme,
+      randomImage: url,
+    }));
+  }
+
+  function handleChange(event) {
+    const { name, value } = event.target;
+    setMeme((prevMeme) => ({
+      ...prevMeme,
+      [name]: value,
+    }));
+  }
+
+  return (
+    <div className="form">
+      <div className="form-input">
+        <input
+          type="text"
+          className="input-top"
+          name="topText"
+          value={meme.topText}
+          onChange={handleChange}
+        />
+        <input
+          type="text"
+          className="input-bottom"
+          name="bottomText"
+          value={meme.bottomText}
+          onChange={handleChange}
```

Comment:
It's essential to guide your users to the action you require them to take.

- [ ] Give the inputs some placeholders